packages/@okta/vuepress-site/terms/index.md

```diff
@@ -6,20 +6,19 @@ title: Developer Terms
 
 showBreadcrumb: False
 
-heading: Terms
+heading: OKTA DEVELOPER EDITION SUBSCRIPTION AGREEMENT
 
 terms:
   - paragraphs:
-    - "OKTA DEVELOPER EDITION SUBSCRIPTION AGREEMENT"
-    - "PLEASE READ THIS DEVELOPER EDITION SUBSCRIPTION AGREEMENT (“AGREEMENT”) CAREFULLY BEFORE USING THE DEVELOPER EDITION SERVICE OFFERED BY OKTA, INC. (“OKTA”). THE TERMS OF THIS AGREEMENT GOVERN YOUR USE OF OKTA’S DEVELOPER EDITION SERVICE. IF YOU DO NOT AGREE TO THESE TERMS, DO NOT USE OKTA’S DEVELOPER EDITION SERVICE. BY ACCEPTING THESE TERMS BELOW, BY USING OKTA’S DEVELOPER EDITION SERVICE IN ANY MANNER, OR BY AGREEING TO AN ORDER FORM, YOU AND THE ENTITY YOU REPRESENT (“CUSTOMER”) AGREE THAT YOU HAVE READ AND AGREE TO BE BOUND BY THE TERMS AND CONDITIONS OF THIS AGREEMENT TO THE EXCLUSION OF ALL OTHER TERMS. IF YOU DO NOT HAVE SUCH AUTHORITY, OR IF YOU DO NOT AGREE WITH THESE TERMS AND CONDITIONS, YOU MUST NOT ACCEPT THIS AGREEMENT AND MAY NOT USE THE DEVELOPER EDITION SERVICE."
+    - "PLEASE READ THIS DEVELOPER EDITION SUBSCRIPTION AGREEMENT (“AGREEMENT”) CAREFULLY BEFORE USING THE DEVELOPER EDITION SERVICE OFFERED BY OKTA, INC. (“OKTA”). THE TERMS OF THIS AGREEMENT GOVERN YOUR USE OF OKTA’S DEVELOPER EDITION SERVICE. IF YOU DO NOT AGREE TO THESE TERMS, THEN DO NOT USE OKTA’S DEVELOPER EDITION SERVICE. BY CREATING A DEVELOPER ACCOUNT, ACCEPTING THESE TERMS IN ANY FASHION, USING OKTA’S DEVELOPER EDITION SERVICE IN ANY MANNER OR BY AGREEING TO AN ORDER FORM, YOU AND THE ENTITY YOU REPRESENT (“CUSTOMER”) AGREE THAT YOU HAVE READ AND AGREE TO BE BOUND BY THIS AGREEMENT TO THE EXCLUSION OF ALL OTHER TERMS. IF YOU DO NOT HAVE SUCH AUTHORITY OR IF YOU DO NOT AGREE WITH THESE TERMS AND CONDITIONS, YOU MUST NOT ACCEPT THIS AGREEMENT AND MAY NOT USE THE DEVELOPER EDITION SERVICE."
```

Comment:
@albertchen-okta There may be legal reasons for this, but does this opening paragraph need to be in all-caps? I find it difficult to read.

Reply:
I'm not sure the legal reason but i just tried to copy the text verbatim in the gdoc (linked the JIRA)

Reply:
per legal "there are certain laws which require certain types of contract terms to be in caps so that they are "conspicuous "